src/server/active_tran_server.cpp

```diff
@@ -247,6 +247,15 @@ active_tran_server::connect_to_page_server (const cubcomm::node &node, const cha
       return ER_NET_PAGESERVER_CONNECTION;
     }
 
+  int exit_code;
```

Comment:
- "exit_code" is usually the name used for a value that is returned by the function or by the program. Use a more intuitive name (e.g. received_code or received_int).
- I think the second condition of failure is "code is not equal CONNECT_ACTIVE_TRAN_TO_PAGE_SERVER.
- Avoid using in the same conditional statement a value that is output by a function.
- To avoid repeating the same sequence of code that sets the error, add a lambda function (use it everywhere possible)
- er_log_debug after er_set is overkill :).

```c
auto ps_conn_error_lambda = [&node] ()
  {
    er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_NET_PAGESERVER_CONNECTION, 1, node.get_host ().c_str ());
    return ER_NET_PAGESERVER_CONNECTION;
  };
  
int returned_code;
if (srv_chn.recv_int (returned_code)!= css_error_code::NO_ERRORS)
  {
    return ps_conn_error_lambda ();
  }
if (returned_code != static_cast<int> (cubcomm::server_server::CONNECT_ACTIVE_TRAN_TO_PAGE_SERVER))
  {
    return ps_conn_error_lambda ();
  }
```